src/patternfly/components/Dropdown/dropdown-multi-select.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="pf-c-dropdown__menu" role="dialog" aria-labelledby="{{id}}" {{#unless pf-c-dropdown--IsExpanded}}hidden="hidden"{{/unless}}>
+    <form class="pf-c-form">
+    <fieldset class="pf-c-form__fieldset">
+        <legend class="pf-u-sr-only">Choose one or more items</legend>
+        {{#> check check--modifier="pf-c-dropdown__menu-item"}}
+            {{#> check-input check-input--attribute='type="checkbox" id="checkbox-apples" name="apples"'}}{{/check-input}}
+            {{#> check-label check-label--attribute='for="checkbox-apples"'}}Apples{{/check-label}}
+        {{/check}}
+        {{#> check check--modifier="pf-c-dropdown__menu-item"}}
+            {{#> check-input check-input--attribute='type="checkbox" id="checkbox-oranges" name="oranges" checked'}}{{/check-input}}
+            {{#> check-label check-label--attribute='for="checkbox-apples"'}}Oranges{{/check-label}}
```

Comment:
The `apples` in this line is a typo:
`check-label--attribute='for="checkbox-apples"'`
Instead it should be `oranges`:
`check-label--attribute='for="checkbox-oranges"'`